### ISSUE TITLE
##[warning]Task 'Check Changed Files' version 1 (ChangedFiles@1) is dependent on a Node version (10) that is end-of-life. Contact the extension owner for an updated version of the task. Task maintainers should review Node upgrade guidance: https://aka.ms/node-runner-guidance

### ISSUE THREAD

**Issue by petetina** [touchifyapp/vsts-changed-files#52]
Hi,
There is a warning in azure pipeline : ##[warning]Task 'Check Changed Files' version 1 (ChangedFiles@1) is dependent on a Node version (10) that is end-of-life. Contact the extension owner for an updated version of the task. Task maintainers should review Node upgrade guidance: https://aka.ms/node-runner-guidance

You should upgrade task.json with this : 
"execution": {
    "Node10": {
      "target": "bash.js",
      "argumentFormat": ""
    },
    "Node20_1": {
      "target": "bash.js",
      "argumentFormat": ""
    }

**Comment by SWarnberg** [comment]
Any news on this?

**Comment by llgjermeni** [comment]
Hello,

Any news on this issue. Does the [Fix node10 deprecated warning from azurepipelines #53](https://github.com/touchifyapp/vsts-changed-files/pull/53) fix the warning? 

Thank you

**Comment by poldenburg** [comment]
@llgjermeni yes, it should fix the warning. We are also waiting for this PR to be merged.
@SomaticIT can you merge the PR and release a new version of the task?

**Comment by SomaticIT** [comment]
Sorry for my late reply on this file.
I rebased and improved your PR and published a new version of the extension that use latest tasklib and node v20.
Thanks for your work.

*The new release is currently being validated and will be available publicly soon.*
